packages/darklang/stdlib/list.dark

```diff
@@ -0,0 +1,21 @@
+module Darklang =
+  module Stdlib =
+    module List =
+
+      let dropLast (list: List<'a>) : List<'a> =
+        match list with
+        | [] -> []
+        | _ ->
+          match (List.length_v0 list) with
+          | 1 -> []
+          | _ ->
```

Comment:
rather than `match`ing on the length like this, can use the `::` match pattern

```fsharp
match list with
| [] -> []
| head :: tail -> 
  ...
```
unless I'm missing something

Reply:
No, you're right, I just wasn't aware that I can use the `::` notation in dark files. Thanks!